src/component/playground/StatusCell/FeatureStatusCell.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Chip } from '@mui/material';
+import { TextCell } from '../../common/Table/cells/TextCell/TextCell';
+import { colors } from '../../../themes/colors';
+import { ReactComponent as FeatureEnabledIcon } from 'assets/icons/isenabled-true.svg';
+import { ReactComponent as FeatureDisabledIcon } from 'assets/icons/isenabled-false.svg';
+
+interface Props {
+    enabled: boolean;
+}
+
+export const FeatureStatusCell = ({ enabled }: Props) => {
+    const bgColor = enabled ? colors.green['100'] : colors.red['200'];
+    const color = enabled ? colors.green['700'] : colors.red['700'];
+
+    const icon = enabled ? (
+        <FeatureEnabledIcon stroke={colors.green['600']} stroke-width="0.25" />
+    ) : (
+        <FeatureDisabledIcon stroke={colors.red['700']} stroke-width="0.25" />
+    );
+    const label = enabled ? 'True' : 'False';
+
+    return (
+        <TextCell>
+            <Chip
+                icon={icon}
+                label={label}
+                color={enabled ? 'success' : 'error'}
+                sx={{
```

Comment:
We should be careful with using the SX property on components that might be rendered many times, such as table cells in tables that might retrieve a lot of results. Rendering performance with the sx prop takes double the time to render compared to a styled component. We should only use sx on components that won't render or re-render a lot of times: 

https://mui.com/system/basics/#performance-tradeoff

Reply:
Didn't know that. Thanks for the heads up